change/@azure-communication-react-43f4bfe2-adde-4bb1-9223-a82eb4a3eb68.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
it looks like changelog items are duplicated

Reply:
Yes......Let me try to remove one of them